openupgrade_scripts/scripts/repair/16.0.1.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Le Filament
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def _compute_repair_price_total(env, model):
+    fees = env[model].search([])
+    fees._compute_price_total_and_subtotal()
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "repair", "16.0.1.0/noupdate_changes.xml")
+    _compute_repair_price_total(env, "repair.fee")
+    _compute_repair_price_total(env, "repair.line")
```

Comment:
This is totally useless.

```suggestion
```

Odoo is already computing these fields:

![image](https://github.com/user-attachments/assets/349be046-d87a-4840-8720-68a7ffcbd34d)

In fact, the improvement will be to do this by SQL instead.

Reply:
Fine I remove it then.
these methods are calling account.tax compute_all() function which is quite complex and I think is better done in python rather than in SQL.